nncf/common/pruning/utils.py

```diff
@@ -385,8 +384,7 @@ def get_num_of_sparse_elements_by_node(node_name: str) -> int:
 
 
 def calculate_in_out_channels_by_masks(pruning_groups: List[Cluster[PrunedLayerInfoBase]],
-                                       masks: Dict[str, NNCFTensor],
-                                       tensor_processor: Type[NNCFCollectorTensorProcessor],
+                                       num_of_sparse_elements_by_node: Dict[str, int],
```

Comment:
Should use `NNCFNodeName` instead of `str`

Reply:
Thanks for the comment! Updated.